apps/web-giddh/src/app/gst/gstR3/gstR3.component.ts

```diff
@@ -68,6 +76,27 @@ export class FileGstR3Component implements OnInit, OnDestroy {
         });
         this.gstFileSuccess$.subscribe(a => this.fileReturnSucces = a);
     }
+    /* Aside pane toggle fixed class */
+    public toggleBodyClass(): void {
+        if (this.asideMenuState === 'in') {
+            document.querySelector('body').classList.add('fixed');
+        } else {
+            document.querySelector('body').classList.remove('fixed');
+        }
+    }
+    /**
+      * This will toggle the settings popup
+      *
+      * @param {*} [event]
+      * @memberof SettingsComponent
```

Comment:
Update @memberof and name of method as it toggles Gst side pane and not settings pane
```suggestion
      * @memberof FileGstR3Component
```

Reply:
Done